tests/Digdir.Domain.Dialogporten.Application.Unit.Tests/Features/V1/ServiceOwner/Dialogs/Commands/CreateDialogTests.cs

```diff
@@ -0,0 +1,93 @@
+using AutoMapper;
+using Digdir.Domain.Dialogporten.Application.Common;
+using Digdir.Domain.Dialogporten.Application.Common.ResourceRegistry;
+using Digdir.Domain.Dialogporten.Application.Common.Services;
+using Digdir.Domain.Dialogporten.Application.Features.V1.ServiceOwner.Dialogs.Commands.Create;
+using Digdir.Tool.Dialogporten.GenerateFakeData;
+using NSubstitute;
+using AuthorizationConstants = Digdir.Domain.Dialogporten.Application.Common.Authorization.Constants;
+
+namespace Digdir.Domain.Dialogporten.Application.Unit.Tests.Features.V1.ServiceOwner.Dialogs.Commands;
+
+public class CreateDialogTests
+{
+    [Fact]
+    public async Task CreateDialogCommand_Should_Return_Forbidden_When_Scope_Is_Missing()
+    {
+        // Arrange
+        var dialogDbContextSub = Substitute.For<Externals.IDialogDbContext>();
+
+        var mapper = new MapperConfiguration(cfg =>
+        {
+            cfg.AddMaps(typeof(CreateDialogCommandHandler).Assembly);
+        }).CreateMapper();
+
+        var unitOfWorkSub = Substitute.For<Externals.IUnitOfWork>();
+        var domainContextSub = Substitute.For<IDomainContext>();
+        var userResourceRegistrySub = Substitute.For<IUserResourceRegistry>();
+        var userOrganizationRegistrySub = Substitute.For<IUserOrganizationRegistry>();
+        var activityServiceSub = Substitute.For<IDialogActivityService>();
```

Comment:
Could potentially extract some of the setup into a setup method to reduce repetition